src/drt/test/aes_nangate45.tcl

```diff
@@ -3,7 +3,7 @@ read_lef Nangate45/Nangate45_tech.lef
 read_lef Nangate45/Nangate45_stdcell.lef
 read_def aes_nangate45_preroute.def
 read_guides aes_nangate45.route_guide
-set_thread_count [expr [exec getconf _NPROCESSORS_ONLN] / 4]
+set_thread_count [expr [cpu_count] / 4]
```

Comment:
Q: is it actually well-defined what happens if `set_thread_count` is called with zero ? Because a machine with 2 cores divided by 4 would result in zero.

Reply:
You get a warning and 1 thread